common/components/ui/Modal.scss

```diff
@@ -0,0 +1,139 @@
+@import "common/sass/variables";
+@import "common/sass/mixins";
+
+$m-background: #fff;
+$m-width-map: (small: 440px, medium: 540px, large: 640px, xlarge: 740px);
+$m-window-padding: 20px;
+$m-header-padding: 15px;
+$m-header-height: 62px;
+$m-content-padding: 20px;
+$m-footer-height: 58px;
+$m-anim-speed: 400ms;
+
+@keyframes modalshade-open {
+  0% {
+    opacity: 0;
+  }
+  70%,
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes modal-open {
+  0%,
+  30% {
+    opacity: 0;
+    transform: translate(-50%, -50%) scale(0.88);
+  }
+  100% {
+    opacity: 1;
+    transform: translate(-50%, -50%) scale(1);
+  }
+}
+
+.Modalshade {
+  position: fixed;
```

Comment:
That filtering is so that if you're inside of a form input in a modal, and you hit escape to blur it, you don't also close the modal.